### ISSUE TITLE
Mac wallet 0.0.3 has stopped displaying Gas

### ISSUE THREAD

**Issue by Javadrinkz** [CityOfZion/neon-wallet#103]
Hi guys.

I signed up to Github specifically to ask this question. I have no coding experience whatsoever, so i'm hoping there's a solution to this issue that doesn't require any of the manual adjustments I see around here. They go completely over my head, like a different language.

With that out of the way, the problem i'm having with my Neo wallet for mac is that the total value in USD has stopped being tracked; it simply stays at the same value, while I can still see the value fluctuating in coinmarketcap.com, the app seems to have lost the ability to track the total value of the account at the moment.

Secondly, I am having the same issue as with the 0.0.2 release (after Bittrex broke Neon). The Gas claim button no longer shows any accumulated Gas.
I did read in another recent thread here that it was suggested to someone that they download the newer "0.0.3c" (the addition to file name being the "c" added on the end).
Thinking this may be the solution, I attempted to download this, however, I was unable to find this download. Now that I think of it, i'm not 100% sure it was even a Mac version that was updated when this newer version was released.

So in a nutshell, I can't see total value, I can't see any accumulation of Gas either.
I'm wondering if anybody can give me some advice to rectify this issue, or perhaps advise if it's a small bug currently being worked on?

Thanks to all

**Comment by Irru-** [comment]
Neon's under maintenance. It'll show up when they're done. Try looking at the Neo subreddit, multiple threads have been made there already.

https://www.reddit.com/r/NEO/comments/6tc5ff/neon_wallet_malfunction_cant_claim_gas_anymore/

**Comment by Javadrinkz** [comment]
Nice. Thanks for the link.
haha, thought i'd ask here so they don't tell me to come here when I ask them!
Thanks for the help.

**Comment by anfn101** [comment]
The wallet is under maintenance, as has been stated on Reddit and our support Slack for a while now. As a reminder (again), **do not use Github as tech support** - this is an area for developers to discuss code. It simply makes it more difficult to work here. You can refer to Reddit or our support Slack, a link to which can be found in the sidebar of the subreddit. **Please let everybody you know know this.** As an aside, I've had to close about 5 other posts on this same exact subject... please read all issues before this one before creating a new one. Thanks.
